templates/jmxtrans.service.epp

```diff
@@ -5,10 +5,18 @@ After=syslog.target network.target
 [Service]
 Type=forking
 User=<%= $user %>
+<% if $jmxtrans::systemd_environment_vars =~ Array[String] { -%>
+<% $jmxtrans::systemd_environment_vars.flatten.each |$var| { -%>
+Environment=<%= $var %>
+<% } -%>
+<% } -%>
+<% if $jmxtrans::working_directory { -%>
```

Comment:
This setting was needed due to when starting the process `/usr/share/jmxtrans/bin/jmxtrans.sh start` it couldn't find the lib file:
```
jmxtrans.sh[26133]: File not found - lib/jmxtrans-all.jar
```

In any case, it has been updated to be a local parameter.